package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "auth0.net",
+  "version": "0.0.1",
```

Comment:
I did not align the version here with the version of the actual SDK as they are not related.